pykusto/_src/expressions.py

```diff
@@ -523,6 +523,13 @@ def has_not(self, exp: StringType, case_sensitive: bool = False) -> '_BooleanExp
             f'{self.as_subexpression()} {"!has_cs" if case_sensitive else "!has"} {_to_kql(exp, True)}'
         ))
 
+    def has_any(self, other: ArrayType) -> '_BooleanExpression':
+        """
+        https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/has-anyoperator
+        """
+        assert isinstance(other, (List, Tuple)), "Compared array must be a list of tabular, scalar, or literal expressions"
```

Comment:
Only a literal list is supported, so why is the declared type `ArrayType`? It should be `Union[List, Tuple]` or something similar. 
Even better, we can use `*other: str`, which indicates more clearly that a list of strings is needed.
Also, this implementation does not support a tabular expression, so let's state that in the documentation.

Reply:
I changed the argument type as advised, and added a matching comment in the operator documentation. I'm not in favor of the `*other: str` option and prefer to stay with the explicit iterable type.